src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App = () => {
+  const getUserById = (id: number) => {
+    const search = usersFromServer.find(usr => usr.id === id) as User;
+
+    if (!search) {
+      throw new Error('user not found');
```

Comment:
Throwing an error when a user is not found might not be the best approach, especially if this is a common occurrence. Consider returning `null` or a default user object instead, and handle this case in the calling code.